api/v2/serializers/details/quota_request.py

```diff
@@ -0,0 +1,161 @@
+        serializer = IdentitySummarySerializer(identity, context=self.context)
+        return serializer.data
+
+    def to_internal_value(self, data):
+        queryset = self.get_queryset()
+        if isinstance(data, dict):
+            identity = data.get("id", None)
+        else:
+            identity = data
+        try:
+            return queryset.get(id=identity)
+        except:
+            raise exceptions.ValidationError(
+                "Identity with id '%s' does not exist."
+                % identity
+            )
+
+class QuotaRelatedField(serializers.PrimaryKeyRelatedField):
+
+    def get_queryset(self):
+        return Quota.objects.all()
+
+    def to_representation(self, value):
+        if value.pk is None:
+            return None
+
+        quota = Quota.objects.get(pk=value.pk)
+        serializer = QuotaSummarySerializer(quota, context=self.context)
+        return serializer.data
+
+    def to_internal_value(self, data):
+        queryset = self.get_queryset()
+        if isinstance(data, dict):
+            identity = data.get("id", None)
+        else:
+            identity = data
+        try:
+            return queryset.get(id=identity)
+        except:
+            raise exceptions.ValidationError(
+                "Quota with id '%s' does not exist."
+                % identity
+            )
+
+class StatusTypeRelatedField(serializers.PrimaryKeyRelatedField):
+
+    def get_queryset(self):
+        return StatusType.objects.all()
+
+    def to_representation(self, value):
+        status_type = StatusType.objects.get(pk=value.pk)
+        serializer = StatusTypeSummarySerializer(status_type, context=self.context)
+        return serializer.data
+
+    def to_internal_value(self, data):
+        queryset = self.get_queryset()
+        if isinstance(data, dict):
+            identity = data.get("id", None)
+        else:
+            identity = data
+
+        try:
+            return queryset.get(id=identity)
+        except:
+            raise exceptions.ValidationError(
+                "StatusType with id '%s' does not exist."
+                % identity
+            )
+
+class QuotaRequestSerializer(serializers.HyperlinkedModelSerializer):
+    uuid = serializers.CharField(read_only=True)
+    created_by = UserRelatedField(read_only=True)
+    user = UserSummarySerializer(source='membership.identity.created_by', read_only=True)
+    identity = IdentityRelatedField(source='membership.identity',
+                                    queryset=Identity.objects.none())
+    provider = ProviderSummarySerializer(source='membership.identity.provider', read_only=True)
+    status = StatusTypeRelatedField(queryset=StatusType.objects.none(),
+                                    allow_null=True,
+                                    required=False)
+    quota = QuotaRelatedField(queryset=Quota.objects.all(),
+                              allow_null=True,
+                              required=True)
```

Comment:
If I understand what you did, you set up `to_internal_value` so that we could send PUT/PATCH requests that followed the form a) `{status_type=1}` or b) `{status_type: {id:1}}`, yes?  It might be worth considering (not needed for this PR) creating our own Serializer that captures that behavior - if that's the approach we want to follow, I'd like to apply it to all serializers.  I think most of the v2 serializers only allow the `{status_type=1}` format for now, but I also know that breaks the DRF forms.  Which I wasn't concerned about considered I Model.objects.None()'d all the querysets so the pages would load in a reasonable time.  Part of me wants to standardize on a single format, either a or b, and make that the standard.  Still debating in my mind, but leaning towards b.  I think I started using the PK serializer for v2 because it was the "easy choice" from what DRF supplied out of the box...